packages/@aws-cdk/aws-ecs/lib/container-definition.ts

```diff
@@ -589,6 +602,56 @@ function renderUlimit(ulimit: Ulimit): CfnTaskDefinition.UlimitProperty {
     hardLimit: ulimit.hardLimit,
   };
 }
+/**
+ * The details of a dependency on another container in the task definition.
+ *
+ * @see https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_ContainerDependency.html
+ */
+export interface ContainerDependency {
+  /**
+   * The container to depend on.
+   */
+  readonly container: ContainerDefinition,
+  /**
+   * The dependency condition of the container. Valid values are ContainerDependencyCondition.START, ContainerDependencyCondition.COMPLETE,
+   * ContainerDependencyCondition.SUCCESS and ContainerDependencyCondition.HEALTHY.
+   *
+   * @default - ContainerDependencyCondition.HEALTHY
```

Comment:
We only use `-` when it's not a "real" value. In your case you are actually providing a value...